catena4460_bsec_ulp/catena4460_bsec_ulp.ino

```diff
@@ -101,7 +105,7 @@ void sensorJob_cb(osjob_t *pJob);
 |
 \****************************************************************************/
 
-static const char sVersion[] = "0.2.1";
+static const char sVersion[] = "0.2.2";
```

Comment:
This is at least 0.3.0.  See discussion of semantic versions at https://semver.com

Reply:
Changed the version to 0.3.0.